README.md

```diff
@@ -59,10 +60,11 @@ See the file [COPYING](COPYING) for more details.
 
 HISTORY
 -------
+XQF was originally written by Roman Pozlevich from Pereslavl-Zalessky, Russia. Over 17 years of existence the project has had the following major developers:
```

Comment:
@skybon:
Since you are there, you can also change this line:
https://github.com/XQF/xqf/blob/master/AUTHORS.md#significant-contributors

`s/Significant contributors/Developers/`

Since ALL contributors are _significant_, and since there is no one contribution (even the minor one) that is not _significant_. I will probably details on this file later some example of contribution each people have made.

And this line:
https://github.com/XQF/xqf/blob/master/AUTHORS.md#other-contributors

`s/Other contributors/Contributors/`

Thank you if you do that too. :smiley: